netkan/LICENSE

```diff
@@ -0,0 +1 @@
+../LICENSE
```

Comment:
Similar question as for `README.md`; why the path, and is this file needed if it doesn't make it into the image?

```
root@2b1c37a4c3a8:~# find . /home/netkan -name LICENSE
/home/netkan/.local/lib/python3.12/site-packages/python_dateutil-2.8.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/smmap-5.0.1.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/gitdb-4.0.11.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/typing_extensions-4.12.2.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/yarl-1.11.1.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/six-1.16.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/attrs-24.2.0.dist-info/licenses/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/pynamodb-6.0.1.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/multidict-6.1.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/jsonpatch-1.33.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/requests-2.32.3.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/certifi-2024.8.30.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/pycparser-2.22.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/discord.py-1.7.3.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/packaging-24.1.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/ruamel.yaml-0.18.6.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/netkan_indexer-1.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/cffi-1.17.1.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/charset_normalizer-3.3.2.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/cryptography-43.0.1.dist-info/license_files/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/PyJWT-2.9.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/boto3-1.35.21.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/PyNaCl-1.5.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/gunicorn-23.0.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/wrapt-1.16.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/chardet-4.0.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/async_timeout-3.0.1.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/GitPython-3.1.43.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/internetarchive-4.1.0.dist-info/LICENSE
/home/netkan/.local/lib/python3.12/site-packages/ruamel.yaml.clib-0.2.8.dist-info/LICENSE
```

Reply:
... yeah, it really does not look like this is how it's meant to work. A copy of this _does_ make it into the image, but it just contains the path, not the actual content of the targeted file:

```
root@2b1c37a4c3a8:~# cat /home/netkan/.local/lib/python3.12/site-packages/netkan_indexer-1.0.dist-info/LICENSE
../LICENSE
```

Surely that's not what we're after?

Reply:
Hmm, that's annoying. PyProject, all the files must live in the path and below for them to be included. Given we don't publish the client anywhere, we could just omit the lines. Otherwise it would be moving the pyproject.toml to the root of the project.